pkg/descheduler/descheduler.go

```diff
@@ -105,7 +105,7 @@ func NewDescheduler(karmadaClient karmadaclientset.Interface, kubeClient kuberne
 
 	eventBroadcaster := record.NewBroadcaster()
 	eventBroadcaster.StartStructuredLogging(0)
-	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})
+	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: kubeClient.CoreV1().Events(metav1.NamespaceNone)})
```

Comment:
Why use `NamespaceNone` instead of `NamespaceAll`?

Reply:
Because I think the `""` here means the cluster scope, not all namespaces.